.github/workflows/ci.yml

```
@@ -13,7 +13,9 @@ jobs:
 
     strategy:
       matrix:
-        python-version: ["3.9", "3.10"]
+        python-version: ["3.9", "3.10", "3.11", "3.12"]
+        # Workaround for actions/setup-python#508
+        dev: ["-dev"]
```

https://github.com/actions/setup-python/pull/414 has now been released as https://github.com/actions/setup-python/releases/tag/v4.6.0 and https://github.com/actions/setup-python/releases/tag/v4 so we can do this:

```suggestion
        python-version: ["3.9", "3.10", "3.11", "3.12"]
```